packages/request-node/src/config.ts

```diff
@@ -119,6 +119,18 @@ export function getGasPriceMin(): BigNumber | undefined {
   return gasPriceMin && BigNumber.from(gasPriceMin);
 }
 
+/**
+ * Get the number of block confirmations to wait before considering a transaction successful
+ * @returns the number of block confirmations
+ */
+export function getBlockConfirmations(): number {
+  return (
+    (argv.blockConfirmations && Number(argv.blockConfirmations)) ||
+    (process.env.BLOCK_CONFIRMATIONS && Number(process.env.BLOCK_CONFIRMATIONS)) ||
+    defaultValues.ethereumStorage.ethereum.blockConfirmations
```

Comment:
this is never set, see at the top of this file

Reply:
Fixed.